src/key.js

```diff
@@ -366,6 +366,27 @@ Key.prototype.getEncryptionKeyPacket = function() {
 };
 
 /**
+ * Encrypts all secret key and subkey packets
+ * @param  {String} passphrase
+ * @return {Boolean} true if all key and subkey packets encrypted successfully
+ */
+Key.prototype.encrypt = function(passphrase) {
+  if (this.isPrivate()) {
+    var keys = this.getAllKeyPackets();
+    for (var i = 0; i < keys.length; i++) {
+      try {
+        keys[i].encrypt(passphrase);
+      } catch (e) {
+        return false;
+      }
```

Comment:
No need for the boolean return value. If the encrypt work then fine, otherwise just throw up the exception and let the developer figure out what he did wrong. Swallowing exceptions is too dangerous here.